integration/client_server_test.go

```diff
@@ -464,7 +476,7 @@ func setupClient(t *testing.T, c args, addr string, cacheDir string, golden stri
 
 	// Setup the output file
 	var outputFile string
-	if *update {
+	if *update || createNewFile {
```

Comment:
Why do we need `createNewFile`? The existing test already creates a temp file for the output that we can use to compare.